index/scorch/scorch.go

```diff
@@ -37,6 +37,14 @@ const Name = "scorch"
 
 const Version uint8 = 1
 
+// UnInvertIndex is implemented by various scorch index implementations
```

Comment:
there is only one scorch index implementation, did you mean various segment implementations? in that case, could this interface definition be moved to index/scorch/segment/segment.go? with the rest of the segment related interfaces?